events/wc_druidic_events.txt

```diff
@@ -24,12 +26,37 @@ character_event = {
 		fury_of_wolf_human_event_trigger = yes
 		
 		NOT = { has_global_flag = ongoing_fury_of_wolf_chain_flag }
-		
-		is_primary_war_defender = yes
+		OR = { 
+			any_courtier_or_vassal = {
+				OR = { 
+					has_artifact = book_of_ur 
+					has_artifact = scythe_of_elune 
+				}
+			}
+			ROOT = {
```

Comment:
`ROOT` isn't needed here since outside scope is `ROOT` as well. `OR` from the line below isn't needed then, since outside operator will be `OR`, not `ROOT`.